src/Model/IModel.ts

```diff
@@ -41,10 +41,12 @@ export interface IModel<T extends IModelData> extends IModelMeta<T>, IModelData,
   forceReload(): void;
   applyUpdates(changes?: Partial<T>, meta?: Partial<IModelMeta<T>>, relationships?: any): IModel<T>;
   initializeNewModel(): void;
+  isFieldDirty(fieldName: keyof T): boolean;
```

Comment:
this was defined in the model and not in the interface, so i added it along with some tests